presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -121,6 +121,9 @@ Property Name                                      Description
                                                    Example: ``thrift://192.0.2.3:9083`` or
                                                    ``thrift://192.0.2.3:9083,thrift://192.0.2.4:9083``
 
+``hive.username``                                  Which user is used to access Hive Metastore.
```

Comment:
Let's call this `hive.metastore.username`.

For the default, leave it blank, like the one for `hive.metastore.uri`. Otherwise, it looks like we will send the literal username `"NONE"`.

Reply:
ok,I will fix it